pkg/kubernetes/client.go

```diff
@@ -32,11 +35,12 @@ func NewKubernetesController(kubeClient kubernetes.Interface, meshName string, s
 
```

Comment:
you'll need the kubernetes controller to accept and populate the ConfigInterface